themes/stanford_basic/src/scss/theme/menu/_admin.scss

```diff
@@ -0,0 +1,8 @@
+@charset 'UTF-8';
+
+.toolbar-menu {
```

Comment:
this works, but put it in the [admin scss file](https://github.com/SU-SWS/stanford_profile/blob/11.x/themes/stanford_basic/src/scss/admin/_menu.scss)

Reply:
Sounds good, updated.